### ISSUE TITLE
Framed wall rendering issue with AE2 Facades

### ISSUE THREAD

**Issue by LindaJuffermans** [XFactHD/FramedBlocks#111]
The AE2 facade is seen as a full block for vanilla cobblestone walls but not for Framed Wall:

![image](https://user-images.githubusercontent.com/31929183/167460556-8c6695cf-2d26-4b34-b326-572ad3e2b19e.png)

forge-1.16.5-36.2.34.jar
FramedBlocks-2.15.1.jar
appliedenergistics2-8.4.7.jar

**Comment by XFactHD** [comment]
I tested this and it turns out this is a bug in AE2.
On the left side I placed the walls first and then the facades on the cable, on the right side I placed the facades first and then the walls (cobblestone wall is vanilla).
![2022-05-11_21 24 57](https://user-images.githubusercontent.com/11262040/167930337-e00725c0-0069-48a5-bde5-12f0f17cc5d2.png)

I'll report this to AE2 because it's also an issue in 1.18, but there is most likely very little chance of this getting fixed in AE2 for 1.16.
